soccer/joystick/Joystick.hpp

```diff
@@ -35,7 +34,7 @@ struct JoystickControlValues {
  */
 class Joystick : boost::noncopyable {
 public:
-    Joystick() : _mutex(QMutex::Recursive){};
+    Joystick(){};
```

Comment:
why not just remove the constructor then or have it as default?

Reply:
ACK.